pkg/deploy/strategy/rolling/rolling.go

```diff
@@ -89,6 +103,9 @@ func (s *RollingDeploymentStrategy) Deploy(deployment *kapi.ReplicationControlle
 		return fmt.Errorf("Couldn't decode DeploymentConfig from deployment %s: %v", deployment.Name, err)
 	}
 
+	params := config.Template.Strategy.RollingParams
+	hookRetryPeriod := time.Duration(*params.UpdatePeriodSeconds) * time.Second
```

Comment:
Hard-coded this to 1 second for now (same as recreate), since we don't yet expose it via the hook API properly.